bin/cli.ml

```diff
@@ -148,6 +148,13 @@ let dry_run =
   in
   named (fun x -> `Dry_run x) Arg.(value & flag & info [ "dry-run" ] ~doc)
 
+let draft =
+  let doc =
+    "Produce a draft release that cannot be merged accidentally and has to be \
```

Comment:
How about we flag this as experimental so people don't expect the behaviour to be too stable for now?

Reply:
done!